zkevm-circuits/src/evm_circuit.rs

```diff
@@ -50,6 +52,16 @@ pub struct EvmCircuitConfig<F> {
     copy_table: CopyTable,
     keccak_table: KeccakTable,
     exp_table: ExpTable,
+
+    // rw permutation config
+    rw_permutation_config: PermutationChipConfig<F>,
+
+    // pi for carry over previous chunk context
+    pi_pre_continuity: Column<Instance>,
+    // pi for carry over previous chunk context
```

Comment:
```suggestion
    // pi for carry over chunk context to the next chunk
```
Is this more accurate?

Reply:
Right! and fixed. Thanks for point out that :)